fontir/src/serde.rs

```diff
@@ -70,6 +72,7 @@ impl From<StaticMetadataSerdeRepr> for StaticMetadata {
             from.axes,
             from.named_instances,
             from.glyph_locations.into_iter().collect(),
+            from.postscript_names,
         )
```

Comment:
Hello again! A wee question, tangential to this PR: we noticed that `misc` is serialised in `StaticMetadata`→`StaticMetadataSerdeRepr` but not deserialised in `StaticMetadataSerdeRepr`→`StaticMetadata`; do we populate it elsewhere, or could this be a bug? 🐛

Reply:
looks like a bug to me

Reply:
technically it's a separate issue, might be worth tackling it separately